cmd/virt-launcher/virt-launcher.go

```diff
@@ -338,6 +338,7 @@ func main() {
 	hookSidecars := pflag.Uint("hook-sidecars", 0, "Number of requested hook sidecars, virt-launcher will wait for all of them to become available")
 	noFork := pflag.Bool("no-fork", false, "Fork and let virt-launcher watch itself to react to crashes if set to false")
 	lessPVCSpaceToleration := pflag.Int("less-pvc-space-toleration", 0, "Toleration in percent when PVs' available space is smaller than requested")
+	minimumPVCReserveBytes := pflag.Uint64("minimum-pvc-reserve-bytes", 131072, "Minimum reserve to keep empty on PVC during auto-provision of disk.img")
```

Comment:
Another way to do this would be passing the value through the `SyncVirtualmachine` RPC call from virt-handler. There we have a `VirtualMachineOptions` flag. The disadvantage may be that the value is then not fixed with the pod creation time. Which may increase the cognitive load when doing changes in that area.

Reply:
Please let me know if you prefer it to be moved to that and I'll see what I can do. In that case I might feel compelled to move the lessPVCSpaceToleration with it since they're sibling features... or maybe moving these can be a follow up improvement.

Reply:
It is fine by me how you did it. I don't see an immediate clear advantage.